plugins/modules/provision_workflow_manager.py

```diff
@@ -45,14 +47,25 @@
         provisioning:
             description:
                 - Specifies whether the user intends to perform site assignment only or full provisioning for a wired device.
-                - Set to 'False' to carry out site assignment only.
-                - Set to 'True' to proceed with provisioning to a site.
+                - Set to 'false' to carry out site assignment only.
+                - Set to 'true' to proceed with provisioning to a site.
             type: bool
             required: false
             default: true
+        force_provisioning:
+            description:
+                - Determines whether to force reprovisioning of a device.
+                - Note that reprovisioning cannot change the device's site assignment.
+                - Applicable only for wired devices.
+                - Set to 'true' to enforce reprovisioning, even if the device is already provisioned.
+                - Set to 'false' to skip provisioning for devices that are already provisioned.
```

Comment:
What is the expected behavior if the customer sets force_provisioning is true and provisioning is false?

Reply:
we will get a output like "Cannot assign a provisioned device to the site. Unprovision the device and then try assigning it to the site."  if we want we can improve the error message